pkg/disttask/framework/storage/table_test.go

```diff
@@ -131,11 +131,15 @@ func TestTaskTable(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, proto.TaskStatePending, task.State)
 	require.Nil(t, task.Error)
+	curTime := time.Unix(time.Now().Unix(), 0)
 	require.NoError(t, gm.FailTask(ctx, id, proto.TaskStatePending, errors.New("test error")))
 	task, err = gm.GetTaskByID(ctx, id)
 	require.NoError(t, err)
 	require.Equal(t, proto.TaskStateFailed, task.State)
 	require.ErrorContains(t, task.Error, "test error")
+	endTime, err := storage.GetTaskEndTimeForTest(ctx, gm, id)
+	require.NoError(t, err)
+	require.GreaterOrEqual(t, endTime, curTime)
```

Comment:
Checking if their difference is smaller than a deviation?

Reply:
```
	require.LessOrEqual(t, endTime.After(curTime), time.Now().After(curTime))

```
add this?

Reply:
Add this?
```
	require.LessOrEqual(t, endTime.Sub(curTime), time.Now().Sub(curTime))

```

Reply:
I mean endTime-curTime < 1s

Reply:
We can't make sure endTime-curTime < 1s